Opening case directory no longer working when a certain case is created

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
There is a problem in one health department where opening the case directory is no longer working.
Netzlink was able to mitigate the problem by deleting a certain case from the database (likely identified based on the error message).
When the health department created the case again the case directory was no longer working again.
Based on the error message it seems to be related to the data sync to SurvNet.

`java.lang.IllegalStateException: Duplicate key S355CD-OMDV7J-P4FTE7-KVLAKDDM (attempted merging values de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate`
<details>
`{"Timestamp":"2022-04-07T12:43:11.862+0200","Level":"INFORMATION","Version":"Payara 5.2021.10","LoggerName":"","ThreadID":"83","ThreadName":"http-thread-pool::http-listener-1(4)","TimeMillis":"1649328191862","LevelValue":"800","LogMessage":"E124311.861|     > java.lang.IllegalStateException: Duplicate key S355CD-OMDV7J-P4FTE7-KVLAKDDM (attempted merging values de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@5b423626 and de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@748ba872) (de.symeda.sormas.ui.SormasErrorHandler.java:80)\njavax.ejb.EJBTransactionRolledbackException: null\n\tat de.symeda.sormas.api.caze._CaseFacade_Wrapper.getIndexList(de/symeda/sormas/api/caze/_CaseFacade_Wrapper.java)\n\tat de.symeda.sormas.ui.caze.CaseGrid.getGridData(CaseGrid.java:36)\n\tat de.symeda.sormas.ui.caze.AbstractCaseGrid.lambda$setLazyDataProvider$7f6cf0df$1(AbstractCaseGrid.java:299)\n\tat com.vaadin.data.provider.CallbackDataProvider.fetchFromBackEnd(CallbackDataProvider.java:137)\n\tat com.vaadin.data.provider.AbstractBackEndDataProvider.fetch(AbstractBackEndDataProvider.java:61)\n\tat com.vaadin.data.provider.DataProviderWrapper.fetch(DataProviderWrapper.java:91)\n\tat com.vaadin.data.provider.DataCommunicator.fetchItemsWithRange(DataCommunicator.java:429)\n\tat com.vaadin.data.provider.DataCommunicator.sendDataToClient(DataCommunicator.java:402)\n\tat com.vaadin.data.provider.DataCommunicator.beforeClientResponse(DataCommunicator.java:364)\n\tat com.vaadin.server.communication.UidlWriter.write(UidlWriter.java:126)\n\tat com.vaadin.server.communication.UidlRequestHandler.writeUidl(UidlRequestHandler.java:125)\n\tat com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:93)\n\tat com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)\n\tat com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)\n\tat com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:750)\n\tat org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)\n\tat de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor784.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor253.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor228.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)\n\tat com.sun.proxy.$Proxy891.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)\n\tat org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)\n\tat org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)\n\tat com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)\n\tat org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)\n\tat org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)\n\tat org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)\n\tat org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)\n\tat org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)\n\tat org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)\n\tat org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)\n\tat org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: javax.transaction.TransactionRolledbackException: CORBA TRANSACTION_ROLLEDBACK 9998 Maybe; nested exception is: \n\torg.omg.CORBA.TRANSACTION_ROLLEDBACK: ----------BEGIN server-side stack trace----------\norg.omg.CORBA.TRANSACTION_ROLLEDBACK:   vmcid: 0x2000  minor code: 1806 completed: Maybe\n\tat org.glassfish.enterprise.iiop.impl.POAProtocolMgr.mapException(POAProtocolMgr.java:356)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2357)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2330)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2173)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandlerDelegate.invoke(EJBObjectInvocationHandlerDelegate.java:79)\n\tat com.sun.proxy.$Proxy679.getIndexList(Unknown Source)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4222.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:143)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:173)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:528)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:199)\n\tat com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1565)\n\tat com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:128)\n\tat com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)\n\tat com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)\n\tat de.symeda.sormas.api.caze.__CaseFacade_Remote_DynamicStub.getIndexList(de/symeda/sormas/api/caze/__CaseFacade_Remote_DynamicStub.java)\n\tat de.symeda.sormas.api.caze._CaseFacade_Wrapper.getIndexList(de/symeda/sormas/api/caze/_CaseFacade_Wrapper.java)\n\tat de.symeda.sormas.ui.caze.CaseGrid.getGridData(CaseGrid.java:36)\n\tat de.symeda.sormas.ui.caze.AbstractCaseGrid.lambda$setLazyDataProvider$7f6cf0df$1(AbstractCaseGrid.java:299)\n\tat com.vaadin.data.provider.CallbackDataProvider.fetchFromBackEnd(CallbackDataProvider.java:137)\n\tat com.vaadin.data.provider.AbstractBackEndDataProvider.fetch(AbstractBackEndDataProvider.java:61)\n\tat com.vaadin.data.provider.DataProviderWrapper.fetch(DataProviderWrapper.java:91)\n\tat com.vaadin.data.provider.DataCommunicator.fetchItemsWithRange(DataCommunicator.java:429)\n\tat com.vaadin.data.provider.DataCommunicator.sendDataToClient(DataCommunicator.java:402)\n\tat com.vaadin.data.provider.DataCommunicator.beforeClientResponse(DataCommunicator.java:364)\n\tat com.vaadin.server.communication.UidlWriter.write(UidlWriter.java:126)\n\tat com.vaadin.server.communication.UidlRequestHandler.writeUidl(UidlRequestHandler.java:125)\n\tat com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:93)\n\tat com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)\n\tat com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)\n\tat com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:750)\n\tat org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)\n\tat de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor784.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor253.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor228.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)\n\tat com.sun.proxy.$Proxy891.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)\n\tat org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)\n\tat org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)\n\tat com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)\n\tat org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)\n\tat org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)\n\tat org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)\n\tat org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)\n\tat org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)\n\tat org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)\n\tat org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)\n\tat org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: javax.ejb.TransactionRolledbackLocalException: Exception thrown from bean: java.lang.IllegalStateException: Duplicate key S355CD-OMDV7J-P4FTE7-KVLAKDDM (attempted merging values de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@5b423626 and de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@748ba872)\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.checkExceptionClientTx(EJBContainerTransactionManager.java:615)\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:485)\n\tat com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4601)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2134)\n\t... 103 more\nCaused by: java.lang.IllegalStateException: Duplicate key S355CD-OMDV7J-P4FTE7-KVLAKDDM (attempted merging values de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@5b423626 and de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@748ba872)\n\tat java.base/java.util.stream.Collectors.duplicateKeyException(Collectors.java:133)\n\tat java.base/java.util.stream.Collectors.lambda$uniqKeysMapAccumulator$1(Collectors.java:180)\n\tat java.base/java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)\n\tat java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)\n\tat java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)\n\tat java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)\n\tat java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)\n\tat java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)\n\tat java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)\n\tat de.symeda.sormas.backend.caze.CaseFacadeEjb.getIndexList(CaseFacadeEjb.java:576)\n\tat de.symeda.sormas.backend.caze.CaseFacadeEjb.getIndexList(CaseFacadeEjb.java:333)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4223.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat de.symeda.sormas.backend.util.ValidationConstraintViolationInterceptor.handleValidationConstraintViolation(ValidationConstraintViolationInterceptor.java:34)\n\tat jdk.internal.reflect.GeneratedMethodAccessor255.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat de.symeda.sormas.backend.util.PerformanceLoggingInterceptor.logInvokeDuration(PerformanceLoggingInterceptor.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor254.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor253.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:81)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor228.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:205)\n\t... 101 more\n\n----------END server-side stack trace----------  vmcid: 0x2000  minor code: 1806 completed: Maybe\n\tat com.sun.corba.ee.impl.javax.rmi.CORBA.Util.mapSystemException(Util.java:277)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:201)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)\n\tat com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)\n\tat de.symeda.sormas.api.caze.__CaseFacade_Remote_DynamicStub.getIndexList(de/symeda/sormas/api/caze/__CaseFacade_Remote_DynamicStub.java)\n\t... 85 common frames omitted\nCaused by: org.omg.CORBA.TRANSACTION_ROLLEDBACK: ----------BEGIN server-side stack trace----------\norg.omg.CORBA.TRANSACTION_ROLLEDBACK:   vmcid: 0x2000  minor code: 1806 completed: Maybe\n\tat org.glassfish.enterprise.iiop.impl.POAProtocolMgr.mapException(POAProtocolMgr.java:356)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2357)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2330)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2173)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandlerDelegate.invoke(EJBObjectInvocationHandlerDelegate.java:79)\n\tat com.sun.proxy.$Proxy679.getIndexList(Unknown Source)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4222.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:143)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:173)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:528)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:199)\n\tat com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1565)\n\tat com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:128)\n\tat com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)\n\tat com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)\n\tat de.symeda.sormas.api.caze.__CaseFacade_Remote_DynamicStub.getIndexList(de/symeda/sormas/api/caze/__CaseFacade_Remote_DynamicStub.java)\n\tat de.symeda.sormas.api.caze._CaseFacade_Wrapper.getIndexList(de/symeda/sormas/api/caze/_CaseFacade_Wrapper.java)\n\tat de.symeda.sormas.ui.caze.CaseGrid.getGridData(CaseGrid.java:36)\n\tat de.symeda.sormas.ui.caze.AbstractCaseGrid.lambda$setLazyDataProvider$7f6cf0df$1(AbstractCaseGrid.java:299)\n\tat com.vaadin.data.provider.CallbackDataProvider.fetchFromBackEnd(CallbackDataProvider.java:137)\n\tat com.vaadin.data.provider.AbstractBackEndDataProvider.fetch(AbstractBackEndDataProvider.java:61)\n\tat com.vaadin.data.provider.DataProviderWrapper.fetch(DataProviderWrapper.java:91)\n\tat com.vaadin.data.provider.DataCommunicator.fetchItemsWithRange(DataCommunicator.java:429)\n\tat com.vaadin.data.provider.DataCommunicator.sendDataToClient(DataCommunicator.java:402)\n\tat com.vaadin.data.provider.DataCommunicator.beforeClientResponse(DataCommunicator.java:364)\n\tat com.vaadin.server.communication.UidlWriter.write(UidlWriter.java:126)\n\tat com.vaadin.server.communication.UidlRequestHandler.writeUidl(UidlRequestHandler.java:125)\n\tat com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:93)\n\tat com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)\n\tat com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)\n\tat com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:750)\n\tat org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)\n\tat de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor784.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor253.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor228.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)\n\tat com.sun.proxy.$Proxy891.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)\n\tat org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)\n\tat org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)\n\tat com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)\n\tat org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)\n\tat org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)\n\tat org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)\n\tat org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)\n\tat org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)\n\tat org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)\n\tat org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)\n\tat org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: javax.ejb.TransactionRolledbackLocalException: Exception thrown from bean: java.lang.IllegalStateException: Duplicate key S355CD-OMDV7J-P4FTE7-KVLAKDDM (attempted merging values de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@5b423626 and de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@748ba872)\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.checkExceptionClientTx(EJBContainerTransactionManager.java:615)\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:485)\n\tat com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4601)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2134)\n\t... 103 more\nCaused by: java.lang.IllegalStateException: Duplicate key S355CD-OMDV7J-P4FTE7-KVLAKDDM (attempted merging values de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@5b423626 and de.symeda.sormas.backend.share.ExternalShareInfoCountAndLatestDate@748ba872)\n\tat java.base/java.util.stream.Collectors.duplicateKeyException(Collectors.java:133)\n\tat java.base/java.util.stream.Collectors.lambda$uniqKeysMapAccumulator$1(Collectors.java:180)\n\tat java.base/java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)\n\tat java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)\n\tat java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)\n\tat java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)\n\tat java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)\n\tat java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)\n\tat java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)\n\tat de.symeda.sormas.backend.caze.CaseFacadeEjb.getIndexList(CaseFacadeEjb.java:576)\n\tat de.symeda.sormas.backend.caze.CaseFacadeEjb.getIndexList(CaseFacadeEjb.java:333)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4223.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat de.symeda.sormas.backend.util.ValidationConstraintViolationInterceptor.handleValidationConstraintViolation(ValidationConstraintViolationInterceptor.java:34)\n\tat jdk.internal.reflect.GeneratedMethodAccessor255.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat de.symeda.sormas.backend.util.PerformanceLoggingInterceptor.logInvokeDuration(PerformanceLoggingInterceptor.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor254.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor253.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:81)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor228.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:205)\n\t... 101 more\n\n----------END server-side stack trace----------\n\tat jdk.internal.reflect.GeneratedConstructorAccessor8277.newInstance(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\tat com.sun.corba.ee.impl.protocol.giopmsgheaders.MessageBase.getSystemException(MessageBase.java:813)\n\tat com.sun.corba.ee.impl.protocol.giopmsgheaders.ReplyMessage_1_2.getSystemException(ReplyMessage_1_2.java:131)\n\tat com.sun.corba.ee.impl.protocol.MessageMediatorImpl.getSystemExceptionReply(MessageMediatorImpl.java:610)\n\tat com.sun.corba.ee.impl.protocol.ClientRequestDispatcherImpl.processResponse(ClientRequestDispatcherImpl.java:506)\n\tat com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:166)\n\tat com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)\n\t... 88 common frames omitted\n"}`
</details>

### Steps to Reproduce
<!-- Optional; please add more steps if necessary -->
Unclear.

### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* SORMAS version: 1.69 (?)
* Server URL: Production DE

### Additional Information
<!-- Optional -->

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/share/ExternalShareInfoService.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/share/ExternalShareInfoService.java
@@ -161,6 +161,7 @@ public class ExternalShareInfoService extends AdoServiceWithUserFilter<ExternalS
 		countSubQuery.groupBy(countAssociatedObject);
 
 		cq.multiselect(associatedObjectUuid, countSubQuery, creationDate, root.get(ExternalShareInfo.STATUS));
+		cq.distinct(true);
 		cq.where(
 			cb.function(ExtendedPostgreSQL94Dialect.CONCAT_FUNCTION, String.class, associatedObjectId, creationDate).in(latestShareInfoSubQuery),
 			associatedObjectId.in(ids));

